examples/ssx-test-token-gated/README.md

```diff
@@ -0,0 +1,338 @@
+# SSX ENS Token Gated + RainbowKit
+
+This example will show developers how to enable token-gated access in their dapp with SSX. Additionally, it will show a developer how to integrate and use RainbowKit in an SSX-powered dapp.
+
+## Create the dapp
+
+The initial setup will be done using SSX's dapp creation tool. Type the following in your terminal to get started:
+
+```bash
+yarn create @spruceid/ssx-dapp token-gated-example
+```
+
+For this example we will be using the following options in the setup tool:
+
+- Typescript
+- Leave the other options empty
+
+## Setup Alchemy Account
+
+An Alchemy account and API key will be necessary since alchemy-sdk is used to resolve tokens owned by an account.
+To configure the key in the project create a `.env.development` file and add the key like the following:
+
+```bash
+# token-gated-example/.env
+REACT_APP_ALCHEMY_API_KEY=YOUR_KEY
+```
+
+## Setup Alchemy SDK
+
+The SDK dependency can be installed with the following command:
+
+```bash
+yarn add alchemy-sdk
+```
+
+## Setup RainbowKit
+
+[RainbowKit](https://www.rainbowkit.com/) is used in this example. Add the required dependency by typing the following in your terminal:
+
+```bash
+yarn add @rainbow-me/rainbowkit wagmi
+```
+
+Additionally, you will also need to add the `ssx-react` dependency. To add it, type the following in the terminal:
+
+```bash
+yarn add @spruceid/ssx-react
+```
+
+Head to `src/index.tsx` and add the following:
+
+```tsx
+/** src/index.tsx **/
+
```

Comment:
Define where to add this code. 
Confusing whether I need to delete the whole code in the file or add the below code to existing code.